slimta/envelope.py

```diff
@@ -24,19 +24,32 @@
 
 """
 
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
 
 import re
 import copy
-import cStringIO
 from email.message import Message
 from email.generator import Generator
-from email.parser import Parser, FeedParser
+from email.parser import Parser
+
+import six
+from six.moves import cStringIO
+from six import BytesIO
```

Comment:
This branch does not support Python 2.6 right?

In 2.7, the `BytesIO` class is available in [`io.BytesIO`](https://docs.python.org/2/library/io.html#io.BytesIO), so it shouldn't need to be imported from six.

Also, what's the difference between `cStringIO` and `BytesIO`? In other files, you replaced occurrences of `cStringIO` with `BytesIO`, can that be done here too?